lib/executor/constant_arrival_rate.go

```diff
@@ -227,16 +229,26 @@ func (car ConstantArrivalRate) Run(ctx context.Context, out chan<- stats.SampleC
 	}
 
 	vusFmt := pb.GetFixedLengthIntFormat(maxVUs)
-	fmtStr := pb.GetFixedLengthFloatFormat(arrivalRatePerSec, 2) +
-		" iters/s, " + vusFmt + " out of " + vusFmt + " VUs active"
-
-	progresFn := func() (float64, string) {
+	progresFn := func() (float64, []string) {
 		spent := time.Since(startTime)
 		currentInitialisedVUs := atomic.LoadUint64(&initialisedVUs)
 		vusInBuffer := uint64(len(vus))
-		return math.Min(1, float64(spent)/float64(duration)), fmt.Sprintf(fmtStr,
-			arrivalRatePerSec, currentInitialisedVUs-vusInBuffer, currentInitialisedVUs,
-		)
+		progVUs := fmt.Sprintf(vusFmt+"/"+vusFmt+" VUs",
+			currentInitialisedVUs-vusInBuffer, currentInitialisedVUs)
+		progIters := fmt.Sprintf(
```

Comment:
This seems like it should be a constant, and so, outside of the function? Since if it's constant, there's no need to re-calculate it every 100ms...

Reply:
Agreed. Fixed in latest push.